functions/Get-DbaAgentJobOutputFile.ps1

```diff
@@ -0,0 +1,151 @@
+   on the SERVERNAME instance  
+
+.EXAMPLE
+   Get-DbaAgentJobOutputFile -SqlServer SERVERNAME 
+
+   This will return the paths to the output files for each of the job step of all the Agent Jobs
+   on the SERVERNAME instance   
+
+.EXAMPLE
+   Get-DbaAgentJobOutputFile -SqlServer SERVERNAME,SERVERNAME2 -Jobs 'The Agent Job'
+
+   This will return the paths to the output files for each of the job step of the The Agent Job Job 
+   on the SERVERNAME instance and SERVERNAME2
+
+.EXAMPLE
+   $Servers = 'SERVER','SERVER\INSTANCE1'
+   Get-DbaAgentJobOutputFile -SqlServer $Servers -Jobs 'The Agent Job' -OpenFile 
+
+   This will return the paths to the output files for each of the job step of the The Agent Job Job 
+   on the SERVER instance and the SERVER\INSTANCE1 and open the files if they are available
+
+.EXAMPLE 
+   Get-DbaAgentJobOutputFile -SqlServer SERVERNAME  | Out-GridView
+
+   This will return the paths to the output files for each of the job step of all the Agent Jobs
+   on the SERVERNAME instance and Pipe them to Out-GridView
+
+.EXAMPLE 
+   (Get-DbaAgentJobOutputFile -SqlServer SERVERNAME | ogv -PassThru).FileName | Invoke-Item
+
+   This will return the paths to the output files for each of the job step of all the Agent Jobs
+   on the SERVERNAME instance and Pipe them to Out-GridView and enable you to choose the output
+   file and open it
+   
+.EXAMPLE 
+   Get-DbaAgentJobOutputFile -SqlServer SERVERNAME -Verbose
+
+   This will return the paths to the output files for each of the job step of all the Agent Jobs
+   on the SERVERNAME instance and also show the job steps without an output file
+
+.NOTES
+   AUTHOR - Rob Sewell https://sqldbawithabeard.com
+   DATE - 30/10/2016
+
+    dbatools PowerShell module (https://dbatools.io)
+    Copyright (C) 2016 Chrissy LeMaire
+    This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+    This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.
+
+#>
+[CmdletBinding()]
+param
+(# The Server/instance 
+        [Parameter(Mandatory=$true,HelpMessage='The SQL Server Instance', 
+                   ValueFromPipeline=$true,
+                   ValueFromPipelineByPropertyName=$true, 
+                   ValueFromRemainingArguments=$false, 
+                   Position=0)]
+        [ValidateNotNull()]
+        [ValidateNotNullOrEmpty()]
+        [object]$SqlServer,
+        [Parameter(Mandatory=$false,HelpMessage='SQL Credential', 
```

Comment:
when I pipe a sqlinstance to the function, it's treated as if it's a credential and pops up the get-credential window with the instance as user.